scripts/fetch-iana.js

```diff
@@ -19,7 +20,7 @@ https.get(URL, function onResponse (res) {
     rows.forEach(function (row) {
       var obj = row.reduce(reduceRows, {})
 
-      if (obj.description !== 'Unassigned') {
+      if (obj.description !== 'Unassigned' && obj.value !== '306') {
```

Comment:
I'm kind of on the fence on if this should specifically blacklist _306_ code, or maybe just ones where the message is (unused) or similar. I was originally picturing in my mind the blacklist being against `(Unused)`, but of course there is no other code like it, so it wouldn't really make a difference. Since a reason phrase can be any obs-text, there is nothing wrong with it per-se, so I suppose a blacklist on the specific code is good enough.

Reply:
I'm on the fence too, but I think it serves as self documentation. It's a unique case, so handling it in code as unique seems appropriate. 

The logical question would be, do we want to remove any code that in the future is documented as `(Unused)`? 

I can change it right now if you'd prefer to see us filter out all `(Unused)`, otherwise I think singling out 306 is appropriate.

Reply:
No, sorry, it was just a comment :) not a request for a change. I just after that followed the IANA registration and I think there is no clear (good) way to make such a distinction, so a filter on the code seems appropriate.

Reply:
Since this does not need to be performant or anything and there are two filters, I flipped them so we could add a comment about why 306 is ignore for reference :)